feat(aps): update remote access documentation (#4213)
scaleway/docs-content: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx

  |---------------------------------|---------------|--------------------------------------------|--------------------------|--------------------------------------------------|----------------------------|------------------------|
  | ARD (integrated)                | macOS         | ✅ (except for password)                   | ❌                       | ✅ auto                                           | ⭐⭐⭐                       | ⭐⭐⭐                   |
  | RealVNC                         | Windows, Linux, macOS           | 〰️ (some issues)                          | ❌ (not with free version) | ❌ (manual tunnel) | ❌                         | ⭐ (not recommended)   |
  | Remmina                         | Linux         | ✅ (manual, only from local to server)     | ❌                       | ❌ (manual tunnel)                               | ❌                         | ⭐                    |
  | Devolutions Remote Desktop Manager | Windows, Linux, macOS       | ✅                                        | ❌                       | ✅ (requires additional setup - SSH tunnel)      | ❌                         | ⭐⭐                    |

>   <sup>*</sup> These remote desktop clients don't allow audio streaming. If you need audio streaming, we recommend using [Reemo](/tutorials/access-mac-mini-with-reemo/). You need to configure the connection with Reemo using a VNC connection. We recommend first setup an other VNC connection and using this remote desktop client to then configure Reemo.

Reviewer: ```suggestion
  <sup>*</sup> These remote desktop clients don't allow audio streaming. If you need audio streaming, we recommend using [Reemo](/tutorials/access-mac-mini-with-reemo/). You need to configure the connection with Reemo using a VNC connection. We recommend setting up another VNC connection and using this remote desktop client before configuring Reemo.
```
